core/schema/container.go

```diff
@@ -812,6 +812,7 @@ func (s *containerSchema) build(ctx context.Context, parent *core.Container, arg
 	return parent.Build(
 		ctx,
 		dir.Self,
+		dir.Self,
```

Comment:
Shouldn't we also handle it in this case?

Reply:
I think we originally discussed to handle this only in docker build function to avoid any surprises in other scenarios.

Reply:
I had never used container.Build and they are indeed same. I updated the code to cover that as well. 👍